content/blog/2019-11-12-commcall-dec2019.md

```diff
@@ -0,0 +1,64 @@
+
+As someone with a background in science, not software engineering, [**Steffi LaZerte**](#speakers) will share her experiences using automated testing in R to ensure that packages do what they're supposed to do, on all the operating systems they're supposed to do it on, and that they handle weird stuff gracefully.
+
+[**Rich FitzJohn**](#speakers) will talk about how to make testing much more useful than a chore that one does just because it is “best practice”, focussing on how testing has been useful in his work - refactoring code bases, preventing regressions, and improving how code is written in the first place. He will discuss how testing can help at the heart of processes, particularly in collaborative work, and how automating your testing leaves you with more time and energy to focus on your software, and describe strategies for testing as systems grow more complex, such as using mocking to simulate components.
+
+🎤 See below for speaker bios, resources, and how to join the call.
+
+#### Share your thoughts in the comments!
+- What motivated you to start testing?
+- What do you wish your past-self knew?
+- What are your favorite resources on the topic?
+- What questions do you have?
+
+
+#### Join the Call
+
+🕘 **Thursday, December 5, 10-11 AM PDT / 6-7 PM GMT** ([find your timezone](http://bit.ly/2PLOCte))
+
+☎️ Everyone is welcome. No RSVP needed. You can use https://zoom.us/test to check that you are set up for audio and video.
+
+Join Zoom Meeting https://zoom.us/j/297967525. To join by phone, [find your local number](https://zoom.us/u/acO2ayYceg). Meeting ID: 297 967 525.
+
+🎥 After the Call, we’ll post the video and collaborative notes on the [archive page](/commcalls).
+
+#### Resources
+- [Testing](https://devguide.ropensci.org/building.html#testing), Chapter 1.10 in the rOpenSci software development guide
+- [Tests](https://r-pkgs.org/tests.html), Chapter 10 in R Packages, by Hadley Wickham and Jenny Bryan  
+- [An Ode to Testing](/blog/2018/03/13/ode-to-testing/), my first review, by Charles T. Gray
+- [Packages for Testing your R Package](https://itsalocke.com/blog/packages-for-testing-your-r-package/), by Maëlle Salmon on the Locke Data blog
+- [Mocking is catching](https://blog.r-hub.io/2019/10/29/mocking/), on the R-hub blog
+
+
+#### Speakers {#speakers}
+
+<img src="/img/blog-images/2019-11-12-commcall-dec2019/steffi-lazerte.jpg" alt="Steffi LaZerte" style="margin: 0px 20px; width: 200px;" align="left">
+_**Steffi LaZerte** is a consulting R programmer and teacher, and Adjunct Professor at Brandon University in Manitoba. Steffi is the author of the rOpenSci-reviewed weathercan package. She has a PhD in Behavioral Ecology and recently received the Society of Canadian Ornithologists Early Career Research Award._  
+_Steffi on [GitHub](https://github.com/steffilazerte), [Twitter](http://twitter.com/steffilazerte), [Website](http://steffilazerte.ca/), [rOpenSci](/authors/steffi-lazerte/)_  
```

Comment:
Steffi's research career has involved exploring seasonal changes in chipmunk activity patterns, effects of anthropogenic noise on communication in birds, and the development of R packages for behavioural ecology.